tests/test_codex_feed_hooks.py

```
@@ -554,22 +561,26 @@ def test_install_migrates_legacy_codex_hooks_feature(cli_path: str, root: Path)
+    config_toml = config_path.read_text(encoding="utf-8")
+    if config_toml.count("codex_hooks = true") != 1:
+        raise AssertionError(f"codex_hooks should be inserted exactly once: {config_toml!r}")
+    if "# See [features] in the documentation." not in config_toml:
+        raise AssertionError(f"comment with [features] was corrupted: {config_toml!r}")
+    if 'note = "literal [features] mention"' not in config_toml:
+        raise AssertionError(f"string literal with [features] was corrupted: {config_toml!r}")
+
+    lines = config_toml.splitlines()
+    features_index = lines.index("[features]")
+    if lines[features_index + 1] != "codex_hooks = true":
+        raise AssertionError(f"codex_hooks should be inserted into [features]: {config_toml!r}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)